server/src/main/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldMapper.java

```diff
@@ -44,6 +59,8 @@ public class SparseVectorFieldMapper extends FieldMapper {
 
     public static final String CONTENT_TYPE = "sparse_vector";
 
+    public static final NodeFeature SPARSE_VECTOR_STORE_SUPPORT = new NodeFeature("mapper.sparse_vector.store_support");
```

Comment:
Thanks for moving this! This feature should be defined here, but it should be added to the set returned by `MapperFeatures#getFeatures`

Reply:
lol, I am lost, I thought that SparseVectorFieldMapper was in x-pack but it's in server so we have access to the `MapperFeatures`. I'll register the **test** feature there.

Reply:
You're right, a test feature should be fine here. I was on autopilot that new feature == add to `getFeatures`, bug fix == add to `getTestFeatures`.